kv/memdb_buffer.go

```diff
@@ -0,0 +1,80 @@
+package kv
```

Comment:
Updated license. I put my name under PingCAP. Please let me know if it's not appropriate.